templates/title_slide.html.slim

```diff
@@ -24,5 +24,12 @@ section.title(class = role
   - preamble = @document.find_by context: :preamble
   - unless preamble.nil? or preamble.length == 0
     div.preamble=preamble.pop.content
-  - unless author.nil?
-    p.author: small=author
+  - if @document.authors.length > 0
+    p.authors
```

Comment:
We recently revisited the author(s) line in the new HTML 5 converter, see: https://github.com/asciidoctor/asciidoctor/blob/214b20116a7191a01238cbc4006bdec0eee8cea6/lib/asciidoctor/converter/semantic-html5.rb#L127-L142

We should probably generate the same HTML in this converter.

Reply:
Absolutely. Let's take this opportunity to be forward looking. It's not necesssary to cling to the older HTML.

Reply:
We might need to add a few lines of CSS in: https://github.com/asciidoctor/asciidoctor-reveal.js/blob/master/templates/asciidoctor-compatibility.css to make sure that it looks decent by default (i.e., when using the default/built-in style/theme).

Reply:
If this code is already in a separated method and you try to generate semantic HTML5, I think it would be nice to try to reuse the method instead of reimplementing it, what do you think of it ? I'm not for forced reuse everywhere but in this case it seems to make sense.

Reply:
> If this code is already in a separated method and you try to generate semantic HTML5, I think it would be nice to try to reuse the method instead of reimplementing it, what do you think of it ? 
I'm not for forced reuse everywhere but in this case it seems to make sense.

Indeed, this is a good idea and that's one of the reason why the new converter has dedicated methods.

> As the code in the main gem is not yet released maybe we can wait for it since the bugs are opened since a while

You are correct, the new converter is still at an early stage. We want to make it available sooner than later but currently it's not available in any release.
We don't have an exact date for when the new converter will be ready/available so we should not put this feature on hold. 

> or make a temporary copy with a comment indicating it should be deleted later.

Yes, I think it's totally fine to do that 👍🏻

Reply:
> While browsing the code I've found https://github.com/asciidoctor/asciidoctor-reveal.js/blob/master/templates/helpers.rb#L54 , is the comment still relevant 


The comment is still accurate in the sense that the code was copied from https://github.com/jirutka/asciidoctor-html5s/blob/a71db48a1dd5196b668b3a3d93693c5d877c5bf3/data/templates/helpers.rb.

Both `asciidoctor-html5s` and `asciidoctor-reveal.js` are using Slim templates so it's not surprising that they are using similar helper methods.
Having said that, `asciidoctor-reveal.js` has no direct dependency on `asciidoctor-html5s`.

> (if I understand correctly asciidoctor-html5s seems to be subsumed by the new HTML 5 converter inside the main project)?

Not exactly... the goal is similar (i.e., we want to provide an HTML 5 converter that maximizes semantic constructs) but, since `asciidoctor-html5s` is a custom converter, users are free to choose which one they want to use.

Reply:
Thanks for all the explanations!